packages/rs-drive-abci/src/execution/types/execution_operation/mod.rs

```diff
@@ -8,12 +12,54 @@ pub mod signature_verification_operation;
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum ExecutionOperation {
+    DoubleSha256,
```

Comment:
Don't we need Single SHA256 as well?

Reply:
Well we don't use it anywhere right now, so no.